clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -20,9 +26,12 @@ import org.broadinstitute.clio.util.model.UpsertId
   */
 class ElasticsearchIndex[CI <: ClioIndex](
   val clioIndex: CI,
-  private[elasticsearch] val fieldMapper: ElasticsearchFieldMapper
+  val defaults: Json = Json.obj(),
+  private[elasticsearch] val fieldMapper: ElasticsearchFieldMapper,
+  private[elasticsearch] val additionalIdFields: Seq[String] = Seq.empty
```

Comment:
I wonder if we need this, or if we could compute it from `defaults`... get all the keys with non-null values. WDYT?

Reply:
What if we have something defaulted that we don't want in the id?

Reply:
Hopefully we'll never do that, but fair enough.